dlib/dnn/core.h

```diff
@@ -2251,7 +2251,7 @@ namespace dlib
 
         void back_propagate_error(
             const tensor& /*x*/,
-            zero_gradients zero_grads = zero_gradients::yes
+            zero_gradients /*zero_grads*/
```

Comment:
How about `zero_gradients /*zero_grads*/ = zero_gradients::yes`?  So the signature matches all the others still?

Reply:
Fixed, thank you!